src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -2151,6 +2151,35 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     </MSBuild>
 
+    <!--
+        When no specific targets are defined for project references, use GetTargetPath to avoid circular dependencies
+        that can occur when default targets include ResolveProjectReferences.
+        -->
+    <MSBuild
+        Projects="@(_MSBuildProjectReferenceExistent)"
+        Targets="GetTargetPath"
+        BuildInParallel="$(BuildInParallel)"
+        Properties="%(_MSBuildProjectReferenceExistent.SetConfiguration); %(_MSBuildProjectReferenceExistent.SetPlatform);  %(_MSBuildProjectReferenceExistent.SetTargetFramework)"
+        Condition="'%(_MSBuildProjectReferenceExistent.BuildReference)' == 'true' and '@(ProjectReferenceWithConfiguration)' != '' and '$(BuildingInsideVisualStudio)' != 'true' and '$(BuildProjectReferences)' == 'true' and '@(_MSBuildProjectReferenceExistent)' != '' and '%(_MSBuildProjectReferenceExistent.Targets)' == ''"
+        ContinueOnError="$(ContinueOnError)"
+        RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)$(_GlobalPropertiesToRemoveFromProjectReferences)">
+
+      <Output TaskParameter="TargetOutputs" ItemName="_ResolvedProjectReferencePaths"
+        Condition="'%(_MSBuildProjectReferenceExistent.ReferenceOutputAssembly)'=='true' or '$(DesignTimeBuild)' == 'true'" />
+      <Output TaskParameter="TargetOutputs"
+        ItemName="%(_MSBuildProjectReferenceExistent.OutputItemType)"
+        Condition="'%(_MSBuildProjectReferenceExistent.OutputItemType)' != ''" />
+
+    </MSBuild>
+
+      <Output TaskParameter="TargetOutputs" ItemName="_ResolvedProjectReferencePaths"
+        Condition="'%(_MSBuildProjectReferenceExistent.ReferenceOutputAssembly)'=='true' or '$(DesignTimeBuild)' == 'true'" />
+      <Output TaskParameter="TargetOutputs"
+        ItemName="%(_MSBuildProjectReferenceExistent.OutputItemType)"
+        Condition="'%(_MSBuildProjectReferenceExistent.OutputItemType)' != ''" />
+
+    </MSBuild>
```

Comment:
One thing I worry about when duplicating entire blocks like this is that the invocations can get out of sync. @YuliiaKovalova what do you think about this idea:

* instead of providing an explicit separate MSBuild Task invocation with hard-coded GetTargetPath targets, what if we instead computed a default for `_MSBuildProjectReferenceExistent.Targets` and set that on the `_MSBuildProjectReferenceExistent` items, so that we can do just one call to the MSBuild Task?
* If there is concern about modifying the `_MSBuildProjectReferenceExistent` Items, we could potentially create a new Item list for this specific purpose?